Add Elastic Agent 9.3.7 release notes (#15190)
elastic/elastic-agent: docs/release-notes/_snippets/9.3.7/index.md

## 9.3.7 [elastic-agent-release-notes-9.3.7]

### Features and enhancements [elastic-agent-9.3.7-features-enhancements]

> * Adds system.cpu.cores field to self monitoring data. [#14885](https://github.com/elastic/elastic-agent/pull/14885) [#14862](https://github.com/elastic/elastic-agent/issues/14862)

Reviewer: ```suggestion
* Adds `system.cpu.cores` field to self-monitoring data. [#14885](https://github.com/elastic/elastic-agent/pull/14885) [#14862](https://github.com/elastic/elastic-agent/issues/14862)
```

> 

Reviewer: ```suggestion
```

* Set profiles feature gate for OTel profiles pipelines. [#15061](https://github.com/elastic/elastic-agent/pull/15061) 

### Fixes [elastic-agent-9.3.7-fixes]

> * Add missing host.hostname to monitoring-metrics. [#14454](https://github.com/elastic/elastic-agent/pull/14454) 

Reviewer: ```suggestion
* Add missing `host.hostname` to monitoring metrics. [#14454](https://github.com/elastic/elastic-agent/pull/14454) 
```

> * Fix missing beat receiver trace logs from diagnostic bundle. [#15082](https://github.com/elastic/elastic-agent/pull/15082) 

Reviewer: ```suggestion
* Fix an issue where Beat receiver trace logs were missing from the diagnostics bundle. [#15082](https://github.com/elastic/elastic-agent/pull/15082) 
```
